src/main.c

```diff
@@ -286,16 +288,22 @@ int main(int argc, char* args[])
       goto cleanup;
     }
 
+    char buffer[20];
```

Comment:
I'd like this to be toggle-able, so we don't just draw it on every user's display.
Perhaps this is the start of a "debug display"?